From 03390b56eff5b4cc15d135de8a34311c26db259f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 13 Mar 2020 22:05:38 +0100
Subject: [PATCH 25/36] dash: Get content box from the themeNode

Not doing this may lead to using a wrong box area to allocate our children

Fixes #1113
---
 dash.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dash.js b/dash.js
index 989d66d..d2a1f90 100644
--- a/dash.js
+++ b/dash.js
@@ -85,7 +85,7 @@ class DashToDock_MyDashActor extends St.Widget {
 
     vfunc_allocate(box, flags) {
         this.set_allocation(box, flags);
-        let contentBox = box;
+        let contentBox = this.get_theme_node().get_content_box(box);
         let availWidth = contentBox.x2 - contentBox.x1;
         let availHeight = contentBox.y2 - contentBox.y1;
 
-- 
2.25.1

